linera-views/src/scylla_db.rs

```diff
@@ -694,18 +537,18 @@ impl ScyllaDbStoreInternal {
                 for row in rows.into_typed::<(String, String, String, String)>() {
                     let value = row?;
```

Comment:
I still think this would be more readable, despite the helpful comment:
```suggestion
                    let (_, object_kind, name, _) = row?;
```

Reply:
Done. Thank you.